connector/docs/user.md

```diff
@@ -21,40 +21,248 @@ To retrieve information about the user who is currently logged in, use salesforc
-**resetPassword**
+###### resetPassword
 ```xml
 <salesforce.resetPassword configKey="MySFConfig">
-    <userId>user@wso2.com</userId>
+    <userId>0056F000009wCJgQAM</userId>
 </salesforce.resetPassword>
 ```
-**Properties**
+###### Properties
 * userId: The user's Salesforce ID.
-* password: If using setPassword, the new password to assign to the user.
 
-**Related Salesforce documentation**
+###### Sample response
+
+Given below is a sample response for the resetPassword operation.
+
+```xml
+<?xml version='1.0' encoding='utf-8'?>
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns="urn:partner.soap.sforce.com">
+    <soapenv:Header>
+        <LimitInfoHeader>
+            <limitInfo>
+                <current>23</current>
+                <limit>15000</limit>
+                <type>API REQUESTS</type>
+            </limitInfo>
+        </LimitInfoHeader>
+    </soapenv:Header>
+    <soapenv:Body>
+        <resetPasswordResponse>
+            <result>
+                <password>H5fj8A6M</password>
+            </result>
+        </resetPasswordResponse>
+    </soapenv:Body>
+</soapenv:Envelope>
+```
+
+###### Related Salesforce documentation
 
 [https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_setpassword.htm](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_setpassword.htm)
 
+### Sample configuration
+
+Following example illustrates how to connect to Salesforce with the init operation and getUserInfo operation.
+
+1. Create a sample proxy as below :
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="salesforce_getUserInfo"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+   <target>
+      <inSequence>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:loginUrl/text()"
+                   name="loginUrl"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:username/text()"
+                   name="username"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:password/text()"
+                   name="password"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:blocking/text()"
+                   name="blocking"/>
+         <salesforce.init>
+            <loginUrl>{$ctx:loginUrl}</loginUrl>
+            <username>{$ctx:username}</username>
+            <password>{$ctx:password}</password>
+            <blocking>{$ctx:blocking}</blocking>
+         </salesforce.init>
+         <salesforce.getUserInfo/>
+         <respond/>
+      </inSequence>
+      <outSequence>
+         <send/>
+      </outSequence>
+   </target>
+   <description/>
+</proxy>                              
+```
+2. Create a xml file called getUserInfo.xml containing the following xml:
```

Comment:
Create an XML file named getUserInfo.xml and add the XML configurations given below: